packages/smarthr-ui/src/components/Switch/Switch.tsx

```diff
@@ -21,7 +21,8 @@ const switchStyle = tv({
       'supports-[not_selector(:has(+_*))]:shr-border-[revert] supports-[not_selector(:has(+_*))]:shr-bg-[revert]',
 
       // Switch ツマミ
-      'before:shr-box-border before:shr-inline-block before:shr-size-[theme(fontSize.base)] before:shr-scale-[calc(2/3)] before:shr-rounded-full before:shr-border before:shr-border-solid before:shr-border-[theme(colors.grey.30)] before:shr-bg-[theme(colors.grey.30)] before:shr-transition-[transform,margin] before:shr-duration-150 before:shr-ease-out before:shr-content-[""]',
+      'before:shr-box-border before:shr-inline-block before:shr-size-[theme(fontSize.base)] before:shr-scale-[calc(2/3)] before:shr-rounded-full before:shr-border before:shr-border-solid before:shr-border-[theme(colors.grey.65)] before:shr-bg-[theme(colors.grey.65)] before:shr-transition-[transform,margin] before:shr-duration-150 before:shr-ease-out before:shr-content-[""]',
+      'has-[input:disabled:not(:checked)]:before:shr-border-[theme(colors.grey.30)] has-[input:disabled:not(:checked)]:before:shr-bg-[theme(colors.grey.30)]',
```

Comment:
`colors.grey.65` は `textColor.grey`、`colors.grey.30` は `tectColor.disabled` の方が意図が伝わりやすいように思いました!

```suggestion
      'before:shr-box-border before:shr-inline-block before:shr-size-[theme(fontSize.base)] before:shr-scale-[calc(2/3)] before:shr-rounded-full before:shr-border before:shr-border-solid before:shr-border-[theme(textColor.grey)] before:shr-bg-[theme(textColor.grey)] before:shr-transition-[transform,margin] before:shr-duration-150 before:shr-ease-out before:shr-content-[""]',
      'has-[input:disabled:not(:checked)]:before:shr-border-[theme(textColor.disabled)] has-[input:disabled:not(:checked)]:before:shr-bg-[theme(textColor.disabled)]',
```

Reply:
@uknmr 
ご確認ありがとうございます
確かにです、修正しました🙏
https://github.com/kufu/smarthr-ui/pull/5027/commits/eae735358499213aab4233b5715d88686241a623